test/test_cuda.py

```diff
@@ -1563,6 +1563,9 @@ def test_nvtx(self):
         torch.cuda.nvtx.mark("bar")
         torch.cuda.nvtx.range_pop()
 
+    def test_random_neg_values(self):
```

Comment:
skipif `not TEST_CUDA` please!

Reply:
test_cuda is not assuming cuda is available? My bad :/

Reply:
The tests that use the same logic just above are not protected like [this](https://github.com/pytorch/pytorch/blob/aa99aa1cb81fa4f5ff98785fcaba2cb88bedb57f/test/test_cuda.py#L1434-L1435) one.